source/hardware/controllers/denon_mc7000.rst

```diff
@@ -77,326 +50,253 @@ thread.
 User Variables
 ~~~~~~~~~~~~~~
 
-Please check the \*.js mapping file for user variables to:
+Please check the :file:`Denon-MC7000-scripts.js` mapping file for user variables to:
 
 -  activate NeedleDrop sensor while a track is playing (default: false)
--  set the Pitch Fader ranges in % to toggle between them (default: 4,
-   6, 10, 16, 24)
+-  set the Pitch Fader ranges in % to toggle between them
+   (default: 4, 6, 8, 10, 16, 24)
 -  Platter Ring LED mode: single LED on or off (default: 1). Can be
-   switched with SHIFT + Deck button
--  Vinyl Mode on or off at MIXXX start which also triggers the Platter
+   toggled with :hwlabel:`SHIFT` + :hwlabel:`Deck`
+-  Vinyl Mode on or off at Mixxx start which also triggers the Platter
    Ring LED function (default: 1)
 -  Scratch Parameters (default: 33.3, 1/10, 1/10/32)
--  Jog Parameters (default: 30, 3)
+-  Jog Parameters (default: 1, 3)
+
+
+Mixer Section
+~~~~~~~~~~~~~
+
+.. csv-table::
+   :header: "No.", "Control", "Function"
+   :widths: 10 50 250
+
+   "1.", "Input Selector Switch", "Set to the desired audio source from this channel. :hwlabel:`LINE/PHONO` will disable the corresponding Deck (switches off all LEDs and the controller will not send MIDI signals to control Mixxx). This allows the mixer to use an external source. :hwlabel:`USB` will turn the corresponding Deck on and allows to control Mixxx via USB connection."
```

Comment:
This is very hard to read in plain text IMHO.

Reply:
yep. inside tables it is hard to format. This was better in my previous non-tabled view.

Reply:
I found a way that looks quite okay in the rendered page - even though the source looks a bit strange. But as the result is better we should keep it like this.